inst/components/scss/value_box.scss

```diff
@@ -14,78 +110,141 @@
+        margin-top: 0;
+        overflow: visible;
+      }
+
+      @container bslib-value-box (max-width: 300px) {
+        .value-box-grid {
+          .value-box-showcase { // reset padding on showcase_container
+            padding-left: 1rem;
+          }
+        }
       }
     }
-    // value_box(value = )
-    > :nth-child(2) {
-      @include bootstrap-heading($h2-font-size);
-      color: inherit;
-      // add a non-breaking space to ensure it's not 0 height
-      &::after {
-        content: '\00a0 '
+  }
+
+  &.showcase-left-center {
+    .value-box-grid {
+      grid-template-columns: var(---bslib-value-box-showcase-w, 30%) auto;
+    }
+
+    &[data-full-screen="true"] {
+      .value-box-grid {
+        grid-template-columns: var(---bslib-value-box-showcase-w-fs, 1fr) auto;
       }
     }
-    &.border-start {
-      border-color: var(--bslib-value-box-separator-color) !important;
+
+    &:not([data-fill-screen="true"]) {
+      .value-box-grid {
+        .value-box-showcase {
+          grid-area: left;
+        }
+
+        .value-box-area {
+          grid-area: right;
+        }
+      }
     }
   }
 
-  &[data-full-screen="true"] {
+  &.showcase-bottom {
     .value-box-grid {
-      grid-template-columns: var(--bslib-value-box-widths-full-screen);
+      grid-template-columns: 1fr;
+      grid-template-rows: 1fr var(---bslib-value-box-showcase-h, auto);
+      grid-template-areas:
+          "top"
+          "bottom";
+      overflow: hidden;
+
+      .value-box-showcase {
+        grid-area: bottom;
+        padding: 0;
+        margin: 0;
+      }
+
+      .value-box-area {
+        grid-area: top;
+      }
     }
-    .value-box-showcase {
-      max-height: var(--bslib-value-box-max-height-full-screen);
+
+    &[data-full-screen="true"] {
+      .value-box-grid {
+        grid-template-rows: 1fr var(---bslib-value-box-showcase-h-fs, 2fr);
+
+        .value-box-showcase {
+          padding: 1rem;
+        }
+      }
     }
   }
+}
 
-  &:not([data-full-screen="true"]) {
-    .value-box-showcase.showcase-top-right {
-      margin-top: 0;
+@if ($bslib-value-box-enable-border == "never") {
+  .bslib-value-box {
+    border: 0;
+  }
+}
+@else if ($bslib-value-box-enable-border == "auto") {
+  @if $bslib-value-box-enable-shadow {
+    .bslib-value-box {
+      border: 0;
+    }
+  } @else {
+    .bslib-value-box:not(.border-auto) {
+      border: 0;
     }
   }
 }
 
 // Override layout_inline_grid()'s preference to collapse to full-width on mobile (inside the value box)
 @include media-breakpoint-down(sm) {
-  .bslib-value-box .value-box-grid {
-    grid-template-columns: var(--bslib-value-box-widths) !important;
+  .bslib-value-box.showcase-left-center .value-box-grid {
+    grid-template-columns: var(---bslib-value-box-showcase-w, 30%) auto !important;
+  }
+  .bslib-value-box.showcase-top-right .value-box-grid {
+    grid-template-columns: auto var(---bslib-value-box-showcase-w, 50%) !important;
   }
 }
```

Comment:
Could get rid of this?

Reply:
Yes we could: this was about undoing some of `layout_column_wrap()`'s CSS rules that no longer apply now that we aren't using it internally.